ethereum_clients/client_plugins/geth.js

```diff
@@ -1,30 +1,3 @@
-let platform = 'windows'
-let dataDir = `${process.env.APPDATA}/Ethereum`
-
-// Platform specific initialization
-switch (process.platform) {
-  case 'win32': {
-    platform = 'windows'
```

Comment:
platform on windows will now be `win32` instead of `windows` after this change, correct? does that mess anything up? have you tested on windows?

Reply:
oops good catch